Stratis.Bitcoin/Configuration/TextFileConfiguration.cs

```diff
@@ -39,10 +39,11 @@ public TextFileConfiguration(string[] args)
             this.args = new Dictionary<string, List<string>>();
             foreach (string arg in args)
             {
+                // Split on the FIRST "=".
                 string[] splitted = arg.Split('=');
-                if (splitted.Length == 2)
-                    this.Add(splitted[0], splitted[1]);
-                if (splitted.Length == 1)
+                if (splitted.Length > 1)                    
+                    this.Add(splitted[0], string.Join("=", splitted.Skip(1)));
```

Comment:
Can you please add a little comment here to show what kind of value this helps parse?

Reply:
Done.